src/org/omegat/gui/exttrans/IMachineTranslation.java

```diff
@@ -65,7 +66,7 @@ default void setEnabled(boolean enabled) {
      *            text for translation
      * @return translated text, or null if translation impossible
      */
-    String getTranslation(Language sLang, Language tLang, String text) throws Exception;
```

Comment:
We need to keep the old method in the interface as well, since plugins might not be using `BaseTranslate`.